source/administration/version-archive.rst

```diff
@@ -4,6 +4,9 @@ Version Archive
 Mattermost Enterprise Edition
 ------------------------------
 
+Mattermost Enterprise Edition v4.9.0 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v4-8>`_ - `Download <https://releases.mattermost.com/4.9.0/mattermost-4.9.0-linux-amd64.tar.gz>`_
```

Comment:
I notice the changelog is still referring to v4.8 - is this going to be updated once the v4.9 changelog is merged?